src/Illuminate/Queue/Middleware/Skip.php

```diff
@@ -6,7 +6,7 @@
 
 class Skip
 {
-    public function __construct(protected bool $skip = false)
```

Comment:
You might want to narrow this type a little further:
```suggestion
/**
 * @param  (\Closure(): bool)|bool  $skip
 */
public function __construct(protected Closure|bool $skip = false)
```

Reply:
Done